pyomo/contrib/parmest/tests/test_parmest.py

```diff
@@ -33,6 +33,10 @@
 pynumero_ASL_available = AmplInterface.available()
 testdir = this_file_dir()
 
+# Set the global seed for reproducibility
+_RANDOM_SEED_FOR_TESTING = 524
+np.random.seed(_RANDOM_SEED_FOR_TESTING)  # Set seed for reproducibility
```

Comment:
You'll probably need to put a `try ... except` around this. You are now making `numpy` a hard requirement.